tests/unit/scripting.tcl

```diff
@@ -314,6 +319,12 @@ start_server {tags {"scripting"}} {
         catch {run_script {return redis.call()} 0} e
         set e
     } {*one argument*}
+   
+    test {EVAL - No arguments to server.call/pcall is considered an error} {
+        set e {}
+        catch {run_script {return server.call()} 0} e
+        set e
+    } {*one argument*}
```

Comment:
```suggestion
    test {EVAL - No arguments to server.call/pcall is considered an error} {
        assert_error {run_script {return server.call()} 0} {*one argument*}
    }
```
You don't have to fix this, but the test you copied from was written so badly.

Reply:
yeah, I ran into some other issues just copying the test, so did not dare refactor anything yet :P 
I will change this though.

Reply:
The suggested change does not compile - reverting it. I am going to look at it later as part of TCL refactoring into Python investigation.

Reply:
Yeah, I got the order wrong:
```
assert_error {*one argument*} {run_script {return server.call()} 0} 
```